protos/orchestrator_service.proto

```diff
@@ -681,4 +690,16 @@ message CompleteTaskResponse {
 
 message HealthPing {
 	// No payload
+}
+
+message StreamInstanceHistoryRequest {
+    string instanceId = 1;
+    google.protobuf.StringValue executionId = 2;
+
+    // When set to true, the service may return a more optimized response suitable for workers.
+    bool forWorkItemProcessing = 3;
+}
+
+message HistoryChunk {
```

Comment:
I think this chunk will need to label is these are old (replay) or new events

Reply:
These are always old (replay) events. New events are part of the work item. If the sum of the new events for a work item doesn't fit into the gRPC message size, the backend service should break up the big work item into multiple work items (which are already streamed to the worker).